### PR TITLE
Q1 2020 Commerce & Open Source release (2.3.4, 2.3.3-p1, and 2.2.11)

### PR DESCRIPTION
**Pull request by jeff-matthews**
This pull request (PR) publishes all Q1 2020 related release documentation.

whatsnew
Published all documentation updates related to the Q1 2020 Commerce and Open Source releases.

### CODE REVIEW

File: src/_includes/config/setup-cron_2.3_how-to.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To create the Magento crontab:
 
 Use `--force` to rewrite an existing Magento crontab.
 
- {:.bs-callout-info}
+{:.bs-callout-info}
```
Comment (hguthrie) [inline src/_includes/config/setup-cron_2.3_how-to.md]: The use of this style of note causes the info icon to obscure the bullet point. Not a show-stopper, but looks odd.

File: src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+## Functional fixes
+
+In addition to security enhancements, this release contains the following functional fixes.
+
+### Cart and checkout
+
+<!--- MC-18720  -->
+
+*  Administrators with appropriate but restricted privileges can now view the list of CMS pages at **Content**  >  **Pages**.  Previously, Magento displayed this error: `You cannot define a correlation namestore_table more than once`.
```
Comment (hguthrie) [inline src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md]: Looks like some extra spaces

```suggestion
*  Administrators with appropriate but restricted privileges can now view the list of CMS pages at **Content** > **Pages**. Previously, Magento displayed this error: `You cannot define a correlation namestore_table more than once`.
```

File: src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+### Security enhancements
+
+Twenty-three security enhancements that help close cross-site scripting (XSS) and remote code execution (RCE) vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. Most of these issues require that an attacker first obtains access to the Admin. As a result, we remind you to take all necessary steps to protect your Admin, including but not limited to these efforts: IP whitelisting, two-factor authentication, use of a VPN, the use of a unique location rather than /admin, and good password hygiene.
+
+With this quarterly release, we’ve changed how we describe these security issues.  Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an Adobe Security bulletin. Please see [Security updates available for Magento | APSB20-02](https://helpx.adobe.com/security/products/magento/apsb20-02.html) for more information.
```
Comment (hguthrie) [inline src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md]: There are some spacing problems in this paragraph. Also, there seems to be some strange characters in the link text that causes formatting problems.
 
```suggestion
With this quarterly release, we’ve changed how we describe these security issues. Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an Adobe Security bulletin. Please see [Security updates available for Magento | APSB20-02](https://helpx.adobe.com/security/products/magento/apsb20-02.html)  for more information.
```

File: src/guides/v2.2/release-notes/release-notes-2-2-11-open-source.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+## Functional fixes
+
+In addition to security enhancements, this release contains the following functional fixes.
+
+### Cart and checkout
+
+<!--- MC-18720  -->
+
+*  Administrators with appropriate but restricted privileges can now view the list of CMS pages at **Content**  >  **Pages**.  Previously, Magento displayed this error: `You cannot define a correlation namestore_table more than once`.
```
Comment (hguthrie) [inline src/guides/v2.2/release-notes/release-notes-2-2-11-open-source.md]:
```suggestion
*  Administrators with appropriate but restricted privileges can now view the list of CMS pages at **Content** > **Pages**.  Previously, Magento displayed this error: `You cannot define a correlation namestore_table more than once`.
```

**Comment by jeff-matthews** [comment]
Thanks @hguthrie!

**Comment by jeff-matthews** [comment]
Thanks @meker12!

**Comment by jeff-matthews** [comment]
We can't use the typical review features on protected branches, so I'll need to make your changes in a separate PR @hguthrie & @meker12.

File: src/_includes/config/setup-cron_2.3_how-to.md (hunk 1 above)
Comment (jeff-matthews) [reply src/_includes/config/setup-cron_2.3_how-to.md]: Noted

File: src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md (hunk 2 above)
Comment (jeff-matthews) [reply src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md]: Fixed in #6470

File: src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md (hunk 3 above)
Comment (jeff-matthews) [reply src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md]: Fixed in #6470

File: src/guides/v2.2/release-notes/release-notes-2-2-11-open-source.md (hunk 4 above)
Comment (jeff-matthews) [reply src/guides/v2.2/release-notes/release-notes-2-2-11-open-source.md]: Fixed in #6470

File: src/_includes/config/setup-cron_2.3_how-to.md [hunk 5]
```diff
@@ -44,3 +44,17 @@ content='The `update/cron.php` file exists in [Composer](https://glossary.magent
 
 In [Composer-based installations](https://glossary.magento.com/composer), Magento creates the `update/` directory when you run `composer create-project`. Running `composer install` does not create the `update/` directory (if it did not exist before). See [Recreate the Magento updater](https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html#recreate-magento-updater).'
 %}
+
+As of Magento 2.3.4, mentioned crons have been grouped to `all` (`cron:run`, `update/cron.php`, `setup:cron:run` crons) and `non-optional` (`cron:run` cron only) groups.
```
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/_includes/config/setup-cron_2.3_how-to.md]: Fixed in #6470

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
Need to merge #6471 and re-run tests. Merging will dismiss @meker12 's approval so I'll need another in ~15 minutes.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**